paymentsheet/src/main/java/com/stripe/android/customersheet/CustomerAdapterResult.kt

```diff
@@ -0,0 +1,97 @@
+package com.stripe.android.customersheet
+
+import androidx.annotation.RestrictTo
+import com.stripe.android.core.exception.StripeException
+
+@ExperimentalCustomerSheetApi
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+sealed interface CustomerAdapterResult<T> {
+
+    @ExperimentalCustomerSheetApi
+    @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
```

Comment:
Nit: If we’re not intending to expose these, let’s make them `internal`.

Reply:
We are planning on exposing them, since merchants will need to construct these in the providers for `CustomerAdapter`.